scripts/load_model.sh

```diff
@@ -11,8 +11,8 @@
 set -eo pipefail
 
 # Args
-GIT_REPO_PATH=${1:-ncpi-fhir/ncpi-model-forge}
```

Comment:
Why not allow this to be passed in as an arg so that the script could be used to load other FHIR models?